pkg/controllers/datastoresyncer/datastoresyncer_suite_test.go

```diff
@@ -46,6 +48,7 @@ const (
 	otherClusterID  = "west"
 	localNamespace  = "submariner"
 	brokerNamespace = "submariner-broker"
+	nodeName        = "raiders"
```

Comment:
Raiders fan? :football: :wink:

Reply:
yup - after '03 I used to be embarrassed to say that but they've been more respectable lately.